packages/common/src/groups/HubGroups.ts

```diff
@@ -113,6 +114,15 @@ export async function createHubGroup(
     hubGroup.typeKeywords,
     hubGroup.isDiscussable
   );
+
+  // create or update entity settings
+  const entitySetting = await createOrUpdateEntitySettings(
+    hubGroup,
+    requestOptions
```

Comment:
I'm not sure if this will actually work, since `createOrUpdateEntitySettings` takes an `IHubRequestOptions`, but the `requestOptions` here is an `IUserRequestOptions`.